tools/db_stress.cc

```diff
@@ -382,11 +382,17 @@ DEFINE_bool(use_txn, false,
             "Use TransactionDB. Currently the default write policy is "
             "TxnDBWritePolicy::WRITE_PREPARED");
 
-// Temporarily disable this to allows it to detect new bugs
 DEFINE_int32(compact_files_one_in, 0,
              "If non-zero, then CompactFiles() will be called one for every N "
              "operations IN AVERAGE.  0 indicates CompactFiles() is disabled.");
 
+DEFINE_int32(compact_range_one_in, 0,
+             "If non-zero, then CompactRange() will be called one for every N "
+             "operations IN AVERAGE.  0 indicates CompactRange() is disabled.");
```

Comment:
copy-paste issues:
s/one/once/ 
s/IN AVERAGE/on average/  ?

Reply:
thanks will fix it